usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -3,14 +3,15 @@
 from typing import Dict, Optional
 
 from django.conf import settings
-from elasticsearch_dsl import A, Q as ES_Q
+from elasticsearch_dsl import A
+from elasticsearch_dsl import Q as ES_Q
```

Comment:
Any reason why these are split up?

Reply:
Just a style change. It's easier to read when the `import as` is on a separate line.